### PR TITLE
Diamond Earrings Spritepack

### PR DESCRIPTION
**Pull request by 20RaVen23**
Kaiewe and I made some earrings. I asked him about what name should be fine as author and that was okay with him.

- Added reaction
- JSON
 - [X] Add the needed sprites due some issues I was having, couldn't add them myself.

## Testing:
- [ ] Verify acs look okay
- [ ] Verify earrings selector works properly
- [ ] Verify reaction dialogue is fine

### CODE REVIEW

File: Monika After Story/game/zz_reactions.rpy [hunk 1]
```diff
@@ -2699,6 +2699,27 @@ label mas_reaction_gift_acs_orcaramelo_hairflower_pink:
         $ store.mas_filereacts.delete_file(giftname)
     return
 
+label mas_reaction_gift_acs_RaVen_earrings_diamond:
```
Comment (Booplicate) [inline Monika After Story/game/zz_reactions.rpy:2702]:
```suggestion
label mas_reaction_gift_acs_raven_earrings_diamond:
```

File: spritepacks/released/RaVen & Kai/RaVen_earrings_diamond/mod_assets/monika/j/RaVen_acs_earrings_diamond.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+    "__author": [
+        "RaVen",
+        "Kaiewe"
+    ],
+    "version": 3,
+    "type": 0,
+    "name": "RaVen_earrings_diamond",
+    "img_sit": "RaVen_earrings_diamond",
```
Comment (Booplicate) [inline spritepacks/released/RaVen & Kai/RaVen_earrings_diamond/mod_assets/monika/j/RaVen_acs_earrings_diamond.json:9]:
```suggestion
    "name": "raven_earrings_diamond",
    "img_sit": "raven_earrings_diamond",
```

File: spritepacks/released/RaVen & Kai/RaVen_earrings_diamond/mod_assets/monika/j/RaVen_acs_earrings_diamond.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        "default": "0",
+        "l_default": "5"
+    },
+    "rec_layer": 4,
+    "acs_type": "earrings",
+    "stay_on_start": true,
+    "dlg_plural": true,
+    "select_info": {
+        "display_name": "Earrings (Diamond)",
+        "thumb": "RaVen_earrings_diamond",
```
Comment (Booplicate) [inline spritepacks/released/RaVen & Kai/RaVen_earrings_diamond/mod_assets/monika/j/RaVen_acs_earrings_diamond.json:20]:
```suggestion
        "thumb": "raven_earrings_diamond",
```

File: Monika After Story/game/zz_reactions.rpy [hunk 4]
```diff
@@ -2699,6 +2699,27 @@ label mas_reaction_gift_acs_orcaramelo_hairflower_pink:
...
+    python:
+        sprite_data = mas_getSpriteObjInfo(
+            (store.mas_sprites.SP_ACS, "RaVen_earrings_diamond")
```
Comment (Booplicate) [inline Monika After Story/game/zz_reactions.rpy:2705]:
```suggestion
            (store.mas_sprites.SP_ACS, "raven_earrings_diamond")
```

**Review by Booplicate — CHANGES_REQUESTED** [review]
We use only lowercase characters in var and label names (because it's just more convenient). You'll need to rename the files as well.

**Comment by TriniTDM** [comment]
Those look beautiful...

**Comment by jmwall24** [comment]
Ok this sprite pack doesn't currently work with our hair. Any hairstyle with the right strand that goes in front of her body has the earring go behind that strand, even though that strand starts behind her ear. Will need a fix for this before this spritepack can love forward.

**Comment by multimokia** [comment]
Waiting on #7148

**Comment by 20RaVen23** [comment]
mytimehascome

**Pull request closed without merging**